File: 0389-find-the-difference/solution.java
class Solution {
    public char findTheDifference(String s, String t) {
        int ssum=0,tsum=0;

        for(char c:s.toCharArray()){
            ssum+=c;
        }
        for(char c:t.toCharArray()){
            tsum+=c;
        }
        return (char)(tsum-ssum);
    }
}

File: 0389-find-the-difference/solution.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        ssum=0
        tsum=0

        for i in s:
            ssum+=ord(i)
        
        for i in t:
            tsum+=ord(i)
        
        return chr(tsum-ssum)
